python/src/GaussianProcessFitter_doc.i.in

```diff
@@ -0,0 +1,359 @@
+%feature("docstring") OT::GaussianProcessFitter
+"Algorithm for the evaluation of general linear models.
```

Comment:
You can replace n with \sampleSize, d with \inputDim and p with \outputDim throughout the file.